src/tcgwars/logic/impl/gen4/SupremeVictors.groovy

```diff
@@ -708,8 +706,11 @@ public enum SupremeVictors implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 20
-              if(confirm("Return all [G] Energy attached to $self in order to do 20 additional damage?")) {
-                damage 20
+              if(confirm("Return all [G] Energy attached to $self in order to do aditional damgae?")) {
+                damage 20 * self.cards.filterByEnergyType(G)
```

Comment:
Do the `filterBy` methods return a size by default? Or will this cause an incompatible types exception?